Applications/FileManager/DirectoryView.cpp

```diff
@@ -108,8 +110,32 @@ void DirectoryView::set_view_mode(ViewMode mode)
     ASSERT_NOT_REACHED();
 }
 
+void print_path_history(int index, Vector<String> path_history)
+{
+    if (path_history.size() > 0) {
```

Comment:
Please prefer "early return" style to reduce nesting.
```suggestion
    if (path_history.is_empty())
        return;
```